Doc/whatsnew/3.13.rst

```diff
@@ -581,6 +581,12 @@ Pending Removal in Python 3.15
   All arguments will be removed from :func:`threading.RLock` in Python 3.15.
   (Contributed by Nikita Sobolev in :gh:`102029`.)
 
+* :func:`inspect.getcallargs` has been deprecated since 3.5.
+  Issue a runtime warning and slate it for removal in 3.15.
```

Comment:
```suggestion
* :func:`inspect.getcallargs` has been deprecated since Python 3.5.
  Issue a runtime warning and slate it for removal in Python 3.15.
```